README.md

```diff
@@ -4,66 +4,20 @@
-- [Frontend Happy Hour Picks](https://github.com/FrontEndHappyHour/front-end-happy-hour-picks)
-- [Everything Serverless](https://serverless.css-tricks.com/resources/)
-
-# Tutorials
-- [Colt Steele's Udemy Web Dev Bootcamp](https://www.udemy.com/the-web-developer-bootcamp/)
-- [Mozilla's Web Dev Tutorials](https://developer.mozilla.org/en-US/docs/Learn)
-- [Salesforce Trailhead](https://trailhead.salesforce.com/)
-- [Project Based Tutorials](https://github.com/tuvtran/project-based-learning)
-- [WebGL](https://webglfundamentals.org/)
-- [Workshopper](https://github.com/workshopper)
-- [Intro to React](https://reactjs.org/tutorial/tutorial.html)
-- [Javascript30](https://javascript30.com/)
-- [Google SEO](https://static.googleusercontent.com/media/www.google.com/en//webmasters/docs/search-engine-optimization-starter-guide.pdf)
-
-# Veteran Tech Benefits
-- [AWS Educate Now Available to U.S. Veterans | AWS Government, Education, & Nonprofits Blo](https://aws.amazon.com/blogs/publicsector/aws-educate-now-available-to-u-s-veterans/)
-- [Free Sales force training for Vets](https://veterans.force.com/NewVetForceHome)
-- [Linkedin for Vets](https://socialimpact.linkedin.com/programs/veterans/premiumform)
-
-
 # Coding for Kids
 - [bitsbox](https://bitsbox.com/)
 
-# Web design inspiration
-- [Brutalist Design](http://brutalistwebsites.com/)
-- [Collect UI](https://medium.com/@CollectUI)
-
-# Language Training
-- [Exercism.io](http://exercism.io/)
-
 # Cool Stuff
 - [Experiments with Google](https://experiments.withgoogle.com/)
 
-# Study Materials
-  - [What is Code](https://www.bloomberg.com/graphics/2015-paul-ford-what-is-code/)
-
-  - ## Javascript
-    - [You Don't Know JS](https://github.com/getify/You-Dont-Know-JS)
-    - [Learn Javascript](https://gitbookio.gitbooks.io/javascript/)
-    - [JavaScript for Cats](https://noblemule.gitbooks.io/javascript-for-cats/content/)
-    - [JavaScript Koans](https://github.com/mrdavidlaing/javascript-koans)
-  - ## Computer Science
-    - [Computer Science with Ruby](https://computer-science-one.gitbooks.io/cs101-book/content/)
-    - [Algorithms](http://ressources.unisciel.fr/algoprog/s00aaroot/aa00module1/res/%5BCormen-AL2011%5DIntroduction_To_Algorithms-A3.pdf)
-    - [Base CS](https://medium.com/basecs)
-
-# Stay Up to Date
-- [Hacker News](https://news.ycombinator.com/)
-- [Smashing Magazine](https://www.smashingmagazine.com)
-- [Front End Masters Handbook](https://frontendmasters.com/books/front-end-handbook/2018/)
-- [Javascript Weekly](https://javascriptweekly.com/)
-- [Frontend Focus](https://frontendfoc.us/)
-- [Rand Fishkin at Moz](https://moz.com/community/users/63)
+# Documentation for the web
+  - [All W3C Standards and Drafts](https://www.w3.org/TR/)
+  - [Google's Web Fundamentals](https://developers.google.com/web/fundamentals/)
+  - [MDN Web Docs](https://developer.mozilla.org/en-US/)
+    - [all HTML elements](https://developer.mozilla.org/en-US/docs/Web/HTML/Element)
```

Comment:
Why this not necessary blank space?

Reply:
I thought because it is part of MDN Web Docs, or should I just make it same level with other list?

Reply:
I think nesting is a good idea. I don't think you need an extra line after.